### DISCUSSION TITLE
Globalized luarc

### DISCUSSION THREAD

**Discussion by Z3rio** [LuaLS/LLS-Addons#85] [Q&A] [answered]
Is there some way to set up a globalized version of `.luarc`?

**Comment by Z3rio** [comment]


**Reply by carsakiller** [reply, marked as answer]
You can use the [`--configpath`](https://luals.github.io/wiki/usage/#--configpath) flag to specify the location of a configuration file to use.

**Reply by Z3rio** [reply]
Thank you, I managed to resolve it with that, just took a bit of exploration to see how such flag was passed through mason.nvim (via neovim)
